sanic/asgi.py

```diff
@@ -288,11 +287,22 @@ def __init__(self) -> None:
         """
         Write the response.
         """
-
+        headers = []
+        cookies = dict()
```

Comment:
nit: `{}`

Reply:
@yunstanford Sure. I will update the PR later today and push the changes.

Reply:
@yunstanford Done